101-databricks-workspace-with-custom-vnet-address/azuredeploy.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "workspaceName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the Azure Databricks workspace to create."
+      }
+    },
+    "pricingTier": {
+      "type": "string",
+      "defaultValue": "premium",
+      "allowedValues": ["standard", "premium"],
+      "metadata": {
+        "description": "The pricing tier of workspace."
+      }
+    },
+    "vnetAddressPrefix": {
+      "type": "string",
+      "defaultValue": "10.139",
```

Comment:
We should think about splitting this into two octet params so we can put validation on them (e.g. int parms with a range of 0-255)